packages/boxel-ui/addon/src/components/select/index.gts

```diff
@@ -46,28 +51,18 @@ const BoxelSelect: TemplateOnlyComponent<Signature> = <template>
 
   <style scoped>
     .boxel-select {
-      border: 1px solid var(--boxel-form-control-border-color);
-      border-radius: var(--boxel-form-control-border-radius);
-      background: none;
+      position: relative;
       display: flex;
-      gap: var(--boxel-sp-sm);
-    }
-
-    .boxel-select:after {
-      display: none;
+      align-items: stretch;
+      flex-grow: 1;
+      overflow: hidden;
+      border: 1px solid var(--boxel-border-color);
+      border-radius: var(--boxel-border-radius-sm);
     }
   </style>
-  {{! Note: When editing dropdown of the PowerSelect component we target
-  the styles using :global -- this is not a mistake. The reason we do this is because
-  content is teleported to an element #ember-basic-dropdown-wormhole at the top of the dom
-  Therefore, our out-of-the-box scoped css solution do not work and we must use an escape hatch (ie :global) to
-  ensure styles are being applied correctly.
-   }}
   {{! template-lint-disable require-scoped-style }}
   <style>
     .boxel-select__dropdown {
-      --boxel-select-current-color: var(--boxel-highlight);
-      --boxel-select-selected-color: var(--boxel-light-100);
```

Comment:
If we're removing these css variables, remove them from the usage document as well